shared_model/packages/javascript/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
В сущностиI'm not sure, but I added it because in fact *iroha-lib* is a separate project and can be distributed apart from Iroha by NPM. It's common practice for packages published by NPM, but in the known to me cases, such packages have own separate repo. 
I can delete it if you wanna, don't think that this is critical.

Reply:
I guess we need it, why no.